core/src/main/java/hudson/slaves/SlaveComputer.java

```diff
@@ -150,7 +150,7 @@ private OutputStream decorate(OutputStream os) {
             try {
                 os = f.decorateLogger(this,os);
             } catch (IOException|InterruptedException e) {
-                LOGGER.log(Level.WARNING, "Failed to filter log with "+f, e);
+                logger.log(Level.WARNING, "Failed to filter log with "+f, e);
```

Comment:
p s f must contain upper case, then you should keep `LOGGER` personally i like call it `LOG` as it more compact